src/app/pages/LiveRadioPage/index.jsx

```diff
@@ -16,9 +17,15 @@ const StyledGelPageGrid = styled(GelPageGrid)`
 `;
 
 const LiveRadioPage = ({ pageData }) => {
-  const { language, name, summary, content } = pageData;
-  const blocks = path(['blocks'], content);
-  const { dir } = useContext(ServiceContext);
+  const {
+    language,
+    name,
+    summary,
+    heading,
+    bodySummary,
```

Comment:
What's the difference between `summary` & `bodySummary`? Is it worth just renaming to `body`?

Reply:
Yea there's a couple of things here - `name` == `heading`, but again - maybe something for investigation in the next swarm?